### ISSUE TITLE
`cf buildpacks` should report whether buildpacks are "ready"

### ISSUE THREAD

**Issue by cwlbraa** [cloudfoundry/cli#1952]
**What's the user value of this feature request?**
This would help developers be aware of admin-managed buildpacks that can't actually be used at stage time. 

**Who is the functionality for?**
App devs, mostly, as they may be unaware of admin-managed buildpacks left in unready states. Also useful for platform engineers as it'd provide a clearer indication that a given buildpack may not be consumable by appdevs.

**Who else is affected by the change?**
If people are making bad assumptions about the stability of `cf buildpacks` text output, this might be backwards-incompatible. 

**Is your feature request related to a problem? Please describe.**
https://github.com/cloudfoundry/cloud_controller_ng/issues/1625

**Describe the solution you'd like**
```
position   name                    stack        enabled   locked  state   filename
1          staticfile_buildpack    cflinuxfs3   true      false   READY   staticfile_buildpack-cflinuxfs3-v1.5.7.zip
2          staticfile_buildpack2   cflinuxfs3   true      false   AWAITING_UPLOAD   staticfile_buildpack-cflinuxfs3-v1.5.7.zip
```

**Describe alternatives you've considered**
We have considered not displaying unready buildpacks, but that might make it harder for platform operators to debug failed buildpack uploads.

**Additional context**
https://github.com/cloudfoundry/cloud_controller_ng/issues/1625

@matt-royal

**Comment by johha** [comment]
@cwlbraa Was this ever merged in? I tried this with `cf version 8.4.0+b4c083e.2022-06-02` but there is no `state` column.

**Comment by philippthun** [comment]
"Re-opened": #2344
